nestjs/src/gratitudes/gratitudes.service.ts

```diff
@@ -17,7 +17,7 @@ export class GratitudesService {
   ) {}
 
   public async create(authUser: AuthUser, data: CreateGratitudeDto) {
-    if (authUser.id === data.userId) {
+    if (data.userIds.includes(authUser.id)) {
       throw new BadRequestException('You cannot give feedback to yourself');
     }
```

Comment:
I think we should move this check to the UI side, maybe filtering out the current user from the search field?

Reply:
it's fine to keep on the server-side